packages/tupaia-web/src/components/Modal.tsx

```diff
@@ -22,6 +22,9 @@ const Wrapper = styled.div`
   text-align: center;
   overflow-x: hidden;
   padding: 2rem;
+  ::-webkit-scrollbar {
```

Comment:
I think that we should keep the scroll bar visible even though it looks a bit ugly as it helps users see that the modal is scrolable.